root/static/scripts/edit/URLCleanup.js

```diff
@@ -1026,6 +1026,18 @@ const CLEANUPS = {
       return url;
     },
   },
+  'dynamicrangedb': {
+    match: [new RegExp('^(https?://)?dr\\.loudness-war\\.info', 'i')],
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      // Seems to support http only
+      return url.replace(/^https?:\/\//, 'http://');
```

Comment:
Can be simplified further so as to avoid replacing matched string with the same string:
```suggestion
      return url.replace(/^https:/, 'http:');
```